modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -208,19 +239,15 @@ model OnchainTransaction {
   logs              String?
   cumulativeGasUsed String?
   byzantium         Boolean?
-
-  // Channel fields
-  status         String  @default("submitted") // no enums
-  reason         String // no enums
-  error          String?
-  channelAddress String
-  channel        Channel @relation(fields: [channelAddress], references: [channelAddress])
+  status            Int?
```

Comment:
Shouldn't we remove status as it will be in the parent?

Reply:
Ah, wait, this is the int status. The one where we check if `receipt.status === 0`